content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -489,8 +533,8 @@ The price list in the IAP is always displayed in EUR.
 Special Use cases
 ~~~~~~~~~~~~~~~~~
 
-Cancellation process
-********************
+Export invoices
```

Comment:
This section does seem like it's about the cancellation process and not about exporting invoices. Are you sure you didn't mean to change the title of the other "Cancellation process" section after the credit consumption warning block?

Reply:
You've hit the nail on the head :100: 

@dade-odoo  It is indeed, that section after the One Credit warning block, THANKS d:

![image](https://github.com/user-attachments/assets/d4abfed2-f45f-4e62-98b0-31364e9763d2)
![image](https://github.com/user-attachments/assets/866dbe7b-d369-44a9-8af4-d9af0b09a35f)